src/@ionic-native/plugins/firebase-crashlytics/index.ts

```diff
@@ -40,7 +40,7 @@ export class FirebaseCrashlytics extends IonicNativePlugin {
   @Cordova({
     sync: true
   })
-  initialize(): void {
+  initialise(): any {
```

Comment:
Can you provide more informations why you have to change the return type? Does the function return something?

Reply:
Sure thing: `https://github.com/ReallySmallSoftware/cordova-plugin-firebase-crashlytics#how-to-use-it`

```
var crashlytics = FirebaseCrashlytics.initialise();
crashlytics.logException("my caught exception");
```